packages/kit/src/core/adapt/prerender.js

```diff
@@ -183,7 +203,12 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 				if (response_type === OK) {
 					log.info(`${result.status} ${dependency_path}`);
 				} else {
-					error(result.status, dependency_path, path, 'fetched');
+					error({
+						status: result.status,
+						path: dependency_path,
+						referrer: path,
+						referenceType: 'fetched'
+					});
```

Comment:
Changing this and the one at line 185 simplifies how many function signatures we are juggling for dealing with errors (from 2 very similar down to 1). Now, all of them take one argument.